### PR TITLE
Add HTTP/2 Interop tests to be run

### PR DESCRIPTION
**Pull request by carl-mastrangelo**
I tried running this with "./tools/run_tests/run_interop_tests.py --use_docker -l http2 -s all"

Node seemed to fail, but the other seemed okay.

### CODE REVIEW

**Comment by jtattermusch** [comment]
Please let me know once this is ready for review.

**Comment by carl-mastrangelo** [comment]
@jtattermusch PTAL

**Comment by jtattermusch** [comment]
Please add the --http2_interop flag to https://github.com/grpc/grpc/blob/master/tools/jenkins/run_interop.sh.

That way, the test will be run by jenkins automatically and I'll also be able to see the resulting HTML report.

File: tools/run_tests/run_interop_tests.py [hunk 1]
```diff
@@ -613,6 +658,17 @@ def build_interop_image_jobspec(language, tag=None):
                                             server_port,
                                             docker_image=docker_images.get(str(language)))
           jobs.append(test_job)
+          
+    if args.http2_interop:
+      for test_case in _HTTP2_TEST_CASES:
+        if not test_case in http2Interop.unimplemented_test_cases():
```
Comment (jtattermusch) [inline tools/run_tests/run_interop_tests.py]: do you need this line?

**Comment by carl-mastrangelo** [comment]
I was reluctant to add this by default yet, since both cloud-to-prod tests and Go server tests fail currently.

File: tools/run_tests/run_interop_tests.py (hunk 1 above)
Comment (carl-mastrangelo) [inline tools/run_tests/run_interop_tests.py]: Removed

Comment (jtattermusch) [inline tools/run_tests/run_interop_tests.py]: This line is still here.

**Comment by jtattermusch** [comment]
LGTM. After removing the remaining `if not test_case in http2Interop.unimplemented_test_cases():` line and checking nothing is broken, fell free to merge.

As a next step, let's get the http2 suite running on Jenkins as soon as possible.

**Comment by carl-mastrangelo** [comment]
Done. (again)

**Pull request merged**